Src/Base/AMReX_MPMD.H

```diff
@@ -26,18 +26,27 @@ int MyProgId (); //! Program ID
 class Copier
 {
 public:
-    Copier (BoxArray const& ba, DistributionMapping const& dm);
+    Copier ();
+
+    Copier (BoxArray const& ba, DistributionMapping const& dm,
+            bool send_ba = false);
 
     template <typename FAB>
     void send (FabArray<FAB> const& mf, int icomp, int ncomp) const;
 
     template <typename FAB>
     void recv (FabArray<FAB>& mf, int icomp, int ncomp) const;
 
+    BoxArray get_my_ba();
+
+    DistributionMapping get_my_dm();
```

Comment:
```suggestion
    BoxArray const& boxArray() () const;

    DistributionMapping const& DistributionMap () const;
```
We can change the function names for consistency. We use `boArray` and `DistributionMap` a lot in other places.